plugins/fluentd_telemetry_plugin/src/streamer.py

```diff
@@ -270,11 +270,8 @@ def clear_cached_streaming_data(self):
         self.streaming_metrics_mgr = MonitorStreamingMgr()
 
 if __name__ == "__main__":
```

Comment:
please remove the whole main from the streamer.py
not used any more

Reply:
done